src/main/java/uk/gov/hmcts/reform/sendletter/model/in/LetterWithPdfsRequest.java

```diff
@@ -4,11 +4,12 @@
 import io.swagger.annotations.ApiModelProperty;
 import org.hibernate.validator.constraints.NotEmpty;
 
+import java.io.Serializable;
 import java.util.List;
 import java.util.Map;
 import javax.validation.constraints.Size;
 
-public class LetterWithPdfsRequest {
+public class LetterWithPdfsRequest implements Serializable {
```

Comment:
it is highly recommended to declare `static final long serialVersionUID = 42L;` to avoid runtime computation of it

Reply:
done